.github/workflows/Docs.yaml

```diff
@@ -0,0 +1,50 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy API docs content to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["feature/asciidoctor"]
```

Comment:
this needs to be changed to master I guess

Reply:
also it should filter by repository so that it doesn't run on forks

Reply:
right, I has mostly forgot about that! It needs to be changed to master. 

However I think there is no need to filter by repository... actually I see it as a bonus the fact that forks will deploy github Pages (of course to the forked-repo Pages, e.g. my https://f18m.github.io/libzmq/ site and not to the libzmq official Pages, which will be available at https://zeromq.github.io/libzmq/) since this will allow contributors to check their changes and see how they look like (even if it's also very easy to just produce the HTML locally and preview that locally)...